core/computers.go

```diff
@@ -12,6 +12,14 @@ func Max(a uint32, b uint32) uint32 {
 	return b
 }
 
+// Min returns the minimum number between two given
+func Min(a uint32, b uint32) uint32 {
```

Comment:
Isn't there a `Min` function in the math package?

Reply:
Yes it is but only for float type, and I should cast the values from int to float which is more expensive in terms of speed i gas

Reply:
Yes it is but only for float type, and I should cast the values from int to float which is more expensive in terms of speed i ga

Reply:
I added in Jira task EN-4194 for an integration test for this case

Reply:
Just asking, would it be a good idea to name them MinUint32 in case we need it for other types?

Reply:
Yes, I will do this in my next pull request which is already in review